### PR TITLE
added a couple of resources for spanish lcourses list

### PR DESCRIPTION
**Pull request by victorhck**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) 

### Description 

added a couple of link for spanish free course list. One for LaTeX and on for MarkDown

### Why is this valuable (or not)?

There's no links about LaTeX and MarkDown before in the spanish list.

### How do we know it's really free?

are available in internet free for everyone. No accounts, or something like that needed

### For book lists, is it a book? For course lists, is it a course? etc.

Yep, free courses to follow

### Checklist:
- [X] Not a duplicate
- [X] Included author(s) if appropriate
- [X] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-courses-es.md [hunk 1]
```diff
@@ -104,6 +106,13 @@
 * [Curso Spring](https://www.pildorasinformaticas.es/course/curso-spring) - Juan Díaz (Píldoras Informáticas)
 * [Laravel](https://www.pildorasinformaticas.es/course/laravel) - Juan Díaz (Píldoras Informáticas)
 
+### LaTeX
```
Comment (SethiPandi) [inline free-courses-es.md:109]: Add a line break above. There should be two.

File: free-courses-es.md [hunk 2]
```diff
@@ -104,6 +106,13 @@
...
+
+* [Curso no convencional de LaTeX](https://ondiz.github.io/cursoLatex/)
+
```
Comment (SethiPandi) [inline free-courses-es.md:112]: There should be two line breaks here.

File: free-courses-es.md [hunk 3]
```diff
@@ -104,6 +106,13 @@
...
+### MarkDown
+
+* [Tutorial de MarkDown](https://www.markdowntutorial.com/es/)
```
Comment (SethiPandi) [inline free-courses-es.md:115]: MarkDown is spelt with a lowercase D. "Markdown"

File: free-courses-es.md [hunk 4]
```diff
@@ -104,6 +106,13 @@
 * [Curso Spring](https://www.pildorasinformaticas.es/course/curso-spring) - Juan Díaz (Píldoras Informáticas)
 * [Laravel](https://www.pildorasinformaticas.es/course/laravel) - Juan Díaz (Píldoras Informáticas)
 
+### LaTeX
+
+* [Curso no convencional de LaTeX](https://ondiz.github.io/cursoLatex/)
+
+### MarkDown
```
Comment (SethiPandi) [inline free-courses-es.md:113]: "Markdown" not MarkDown

File: free-courses-es.md [hunk 5]
```diff
@@ -104,6 +106,13 @@
...
+
+* [Tutorial de MarkDown](https://www.markdowntutorial.com/es/)
```
Comment (SethiPandi) [inline free-courses-es.md:116]: Add another line break here, there should be two.

**Review by SethiPandi — COMMENTED** [review]
You can check out the contribution guidelines (CONTRIBUTING.md) for more information on formatting and other guidelines for this repository. You can click on the failed check below and head to Travis CI to see exactly what was wrong with your PR and how to fix it!

**Comment by victorhck** [comment]
fixed

**Comment by eshellman** [comment]
@victorhck Thanks, and welcome to free-programming-books!
@SethiPandi thanks for assist

**Pull request merged**